### ISSUE TITLE
Indentation issue with `@` declarations

### ISSUE THREAD

**Issue by ArmorDarks** [brigade/scss-lint#617]
Settings:

``` yaml
  Indentation:
    enabled: true
    allow_non_nested_indentation: true
    character: tab
    width: 1
```

This:

``` scss
@if $ekzo-enable-grid {
	@if $ekzo-enable-responsive-grid {

		@each $breakpoint, $value in map-get-deep($ekzo-breakpoints, breakpoints) {
			@include ekzo-breakpoint($breakpoint) {
				@include ekzo-grid-types('#{$breakpoint}-');
			}
		}

	} // endif
} // endif
```

will throw

```
  2: [W] Indentation:  Line should be indented 0 tabs, but was indented 1 tabs
  4: [W] Indentation:  Line should be indented 0 tabs, but was indented 2 tabs
```

While this won't throw any errors:

```
@if $ekzo-enable-grid {
@if $ekzo-enable-responsive-grid {

@each $breakpoint, $value in map-get-deep($ekzo-breakpoints, breakpoints) {
	@include ekzo-breakpoint($breakpoint) {
		@include ekzo-grid-types('#{$breakpoint}-');
	}
}

} // endif
} // endif
```

I wonder — is it intentional, or something wrong here?

**Comment by sds** [comment]
Unless I'm misunderstanding the report, this appears to be the same issue as reported as #617. Since that issue is more general I'm going to mark this as duplicate of that one.

If you think they are distinct issues, feel free to reopen. Thanks!

**Comment by ArmorDarks** [comment]
Thanks for reply

I can't tell for sure, since I don't know how SCSS Lint treats this case and what it expects.

But for me it looks like completely another case.

In https://github.com/brigade/scss-lint/issues/618 variables declarations were deliberately placed with forced indendation, and it isn't what SCSS Lint expects. I think it's ok for default rule.

However, here we're just nesting `@if` and `@each` declarations inside each other. May be I'm wrong, but it seems to me that by default it's proper to nest them, instead of flat "list", just like in any block-scoped language.

For example, in JavaScript:

``` javascript
if (myVar) {
    for (item of myVar) {
        console.log(item)
    }
}
```

you don't write it like that:

``` javascript
if (myVar) {
for (item of myVar) {
    console.log(item)
}
}
```
